system/src/Grav/Common/Assets.php

```diff
@@ -105,6 +106,8 @@ public function __construct(array $options = array())
      */
     public function config(array $config)
```

Comment:
I'm not sure if assests should be using locator either as they are compiled files. We should probably have static function to initialize its public directory.